src/ModelerApp.vue

```diff
@@ -15,13 +15,15 @@
         </div>
       </b-card-header>
       <b-card-body class="overflow-hidden position-relative p-0 vh-100">
-        <modeler ref="modeler" @set-xml-manager="xmlManager = $event" @validate="validationErrors = $event" @warnings="warnings = $event"/>
+        <modeler ref="modeler" @set-xml-manager="xmlManager = $event" @validate="validationErrors = $event" @warnings="warnings = $event" :decorations="decorations" />
       </b-card-body>
-      <validation-status
+      <validation-status ref="validationStatus"
         :validation-errors="validationErrors"
         :warnings="warnings"
         :xml-manager="xmlManager"
-      />
+      >
+        <component v-for="(component, index) in validationBar" :key="`validation-status-${index}`" :is="component" />
```

Comment:
I left [a comment](https://github.com/ProcessMaker/modeler/pull/1126#discussion_r382108133) about the key, but looks like it was unchanged. Does simplifying it cause errors?

> I think `index` is sufficient for the `key` property: `:key="index"`.

```suggestion
        <component v-for="(component, index) in validationBar" :key="index" :is="component" />
```